.github/twitter.py

```diff
@@ -0,0 +1,50 @@
+import tweepy
+import requests
+import xml.etree.ElementTree as ET
+import os
+import sys
+
+def main():
+    client = tweepy.Client(
+        consumer_key=os.getenv("API_KEY"),
+        consumer_secret=os.getenv("SECRET_KEY"),
+        access_token=os.getenv("ACCESS_TOKEN"),
+        access_token_secret=os.getenv("ACCESS_SECRET")
+    )
+
+    previous_atom = requests.get("https://squ1rrel.dev/atom.xml", allow_redirects=True).text
+    prev_links_to_items = {i.find("link").text.strip(): i for i in ET.fromstring(previous_atom)[0].findall('item')}
+    current_links_to_items = {i.find("link").text.strip(): i for i in ET.parse('_site/atom.xml').getroot()[0].findall('item')}
+
+    new_articles = { k : current_links_to_items[k] for k in set(current_links_to_items) - set(prev_links_to_items) }
+
+    if len(new_articles) > 0:
+        if len(new_articles) == 1:
+            try:
+                link = list(new_articles.keys())[0]
+                author = list(new_articles.values())[0].find("author").find("name").text
+                tag = list(new_articles.values())[0].findall("category")[1].attrib['term'].replace(" ", "")
+                ctf = list(new_articles.values())[0].findall("category")[0].attrib['term'].replace(" ", "")
```

Comment:
Why are spaces being removed for tag/ctf? some ctfs have spaces in the name; the `atom.xml` should have the exact ctf name from their branding

Reply:
This is so hashtags will work - otherwise it will just hashtag the first word in the tag/ctf title. If you would like to get rid of the hashtag we can use the original branding

Reply:
Or we can use underscores instead of spaces to keep the hashtag